plugins/teachplusplus/teachers/updates/version.yaml

```diff
@@ -48,3 +48,6 @@
 1.0.17:
     - 'Updated table teachplusplus_teachers_subjects'
     - builder_table_update_teachplusplus_teachers_subjects_4.php
+1.0.18:
```

Comment:
Super ze ste si skusili aj taketo migracie. Teoreticky ak este ale nemate realne data o ktore nechcete, prist netreba to robit takto, mozete zmenit uplne prvu migraciu.

Ale samozrejme takto je to najlepsi practice ked uz bude mat realne data, takze kludne moze byt

Reply:
Hej, to mame na plane pred deadlinom dat do jednej migracie.